modules/pub_egress/pub_egress_main.py

```diff
@@ -13,15 +13,19 @@ def main() -> None:
     starting_path_index: int = env.int('STARTING_PATH_INDEX')
     out_path: Path = env.path('OUT_PATH')
     out_path_mdp: Path = env.path('OUT_PATH_MDP')
+    out_mdp_sites: Path = env.path('OUT_MDP_SITES')
     err_path: Path = env.path('ERR_PATH')
-    log_level: str = env.log_level('LOG_LEVEL')
+    log_level: int = env.log_level('LOG_LEVEL')
```

Comment:
This doesn't look right. I'm nearly positive all LOG_LEVEL arguments should be strings.